azurelinuxagent/common/protocol/extensions_goal_state_from_vm_settings.py

```diff
@@ -291,7 +291,7 @@ def _parse_extensions(self, vm_settings):
         #             "settingsSeqNo": 0,
         #             "settings": [
         #                 {
-        #                     "protectedSettingsCertThumbprint": "4037FBF5F1F3014F99B5D6C7799E9B20E6871CB3",
+        #                     "protectedSettingsCertThumbprint": "BD447EF71C3ADDF7C837E84D630F3FAC22CCD22F",
```

Comment:
This is part of a global update. See my comment for the changes in the "data" directory